htdocs/install/mysql/migration/12.0.0-13.0.0.sql

```diff
@@ -585,3 +585,5 @@ insert into llx_c_actioncomm (id, code, type, libelle, module, active, position)
 
 ALTER TABLE llx_export_model MODIFY COLUMN type varchar(64);
 
+
+ALTER TABLE llx_facture_fourn CHANGE ref_supplier ref_supplier VARCHAR(180) NULL;
```

Comment:
To increase size of a column, use instead 
ALTER TABLE llx_table MODIFY COLUMN name varchar(180);